xmake/plugins/pack/wix/main.lua

```diff
@@ -0,0 +1,269 @@
+--!A cross-platform build utility based on Lua
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+-- Copyright (C) 2015-present, TBOOX Open Source Group.
+--
+-- @author      A2va
+-- @file        main.lua
+--
+
+import("lib.detect.find_tool")
+import("private.action.require.impl.packagenv")
+import("private.action.require.impl.install_packages")
+
+import(".batchcmds")
+
+-- get the wixtoolset
+function _get_wix()
+
+    -- enter the environments of wix
+    local oldenvs = packagenv.enter("wixtoolset")
```

Comment:
I detect a problem here. For example if the runtime is different between projects wixtoolset will be installed two times, and after `packagenv.enter("wixtoolset")` WIX_EXTENSIONS will contains all the paths where wixtoolset is installed.
This leads to leads to wixtoolset fails with `wix.exe : error WIX0144: The extension 'WixToolset.UI.wixext' could not be found. Checked paths: WixToolset.UI.wixext`
So, is there a way to ignore all of the configs for a particular package?

Reply:
I still don't understand why it was installed twice

Reply:
I forgot to tell you in my first message, but I installed it with `xrepo install wixtoolset`. It installed once with MD runtime config and the other time with MT runtime. I haven't tested letting xmake install it automatically using the `install_packages` function, I wonder which config it will pick. 
If the binary kind is added to the wixtoolet package in xmake-repo, will it ignore the runtime config (or any configuration at all) ?